core/service/service.go

```diff
@@ -188,12 +188,18 @@ func (s *Service) CreateAccount(ctx context.Context, req *request.CreateAccount)
 		&models.User{Email: req.Email, Name: req.Name, Hash: salt},
 		&models.EmailVerification{Email: req.Email, Hash: v.Hash, Expiry: v.Expiry},
 	); err != nil {
-		return nil, fmt.Errorf("failed to create user: %s", err.Error())
+		if strings.Contains(err.Error(), "Keys cannot be empty") {
```

Comment:
is this one of our errors? Go errors should be lowercase for consistency

```suggestion
		if strings.Contains(err.Error(), "keys cannot be empty") {
```

might want to update the source of the error for this